src/test/java/util/StringStudyTest.java

```diff
@@ -0,0 +1,68 @@
+package util;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class StringStudyTest {
+
+    @Test
+    @DisplayName("문자열 1,2 을 ,로 split 했을 때 1과 2로 잘 분리된다.")
+    void string_split() {
+
+        final String sut = "1,2";
+
+        final String[] actual = sut.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?